### PR TITLE
Add Vue recipe

### PR DESCRIPTION
**Pull request by f-a-a**
Following from the v1.7.0 release. I just thought I could contribute a little. Hope it's helpful!

### CODE REVIEW

File: src/i18n/en/docs/recipes.md [hunk 1]
```diff
@@ -81,3 +81,41 @@ Add Start script to `package.json`
...
+yarn add --dev babel-preset-vue
+```
+
+Then make sure the following Babel config is present.
+
+```javascript
+// .babelrc
+{
+  "presets": ["env", "vue"]
+}
```
Comment (devongovett) [inline src/i18n/en/docs/recipes.md]: This babel preset is probably not needed. env support is built-in to parcel automatically

Comment (devongovett) [reply src/i18n/en/docs/recipes.md]: also what is babel-preset-vue for?

Comment (DeMoorJasper) [reply src/i18n/en/docs/recipes.md]: It's for Vue JSX, so I think it shouldn't be necessary as we use vue-compiler-utils before running it through babel

Comment (devongovett) [reply src/i18n/en/docs/recipes.md]: Oh, maybe we should just add auto JSX support for Vue like we have for React/Preact etc.

Comment (devongovett) [reply src/i18n/en/docs/recipes.md]: I didn't realize Vue even used JSX

**Comment by f-a-a** [comment]
@devongovett @DeMoorJasper I've updated the PR to remove the unneeded dependencies.

**Pull request merged**